hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncConnectionImpl.java

```diff
@@ -121,6 +122,8 @@ public class AsyncConnectionImpl implements AsyncConnection {
   private final String metricsScope;
   private final Optional<MetricsConnection> metrics;
 
+  private JvmPauseMonitor pauseMonitor;
```

Comment:
Make this a singleton, instead? In the case of applications opening or pooling multiple connections, we don't need a separate JvmPauseMonitor per connection, just a single one for the whole application.

Reply:
Good idea !

Reply:
Yeah, I think we don't need to stop it when connection is closed, it may be useful to have it running for as long the client application runs.